test/test_ssl.py

```diff
@@ -0,0 +1,31 @@
+import textwrap
+
+from . import utils
+from .template_projects import CTemplateProject
+
+project_with_ssl_tests = CTemplateProject(
+    setup_py_add=textwrap.dedent(r'''
+        import ssl
+        import sys
+
+        if sys.version_info[0] == 2:
+            from urllib2 import urlopen
+        else:
+            from urllib.request import urlopen
+
+        if sys.version_info[0:2] == (3, 3):
+            data = urlopen("https://www.nist.gov")
+        else:
+            context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
+            data = urlopen("https://www.nist.gov", context=context)
+    ''')
```

Comment:
These parts really show off how neat the solution is :-)

Reply:
:)